shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Val.hs

```diff
@@ -13,43 +13,38 @@
 
 -- | This module defines a generalised notion of a "value" - that is, something
 -- with which we may quantify a transaction output.
---
--- This module is intended for qualified import:
--- > import qualified Cardano.Ledger.Val as Val
 module Cardano.Ledger.Val
   ( Val (..),
-    (<+>),
-    (<->),
-    (<×>),
+    scale,
     invert,
     sumVal,
     scaledMinDeposit,
-
-    -- * Re-exports
-    (Data.PartialOrd.<=),
-    (Data.PartialOrd.>=),
-    (Data.PartialOrd.==),
-    (Data.PartialOrd./=),
-    (Data.PartialOrd.>),
-    (Data.PartialOrd.<),
-    Data.PartialOrd.compare,
   )
 where
 
-import Data.Group (Abelian, invert)
-import Data.PartialOrd hiding ((==))
-import qualified Data.PartialOrd
+import Data.Group (Abelian)
 import Shelley.Spec.Ledger.Coin (Coin (..))
 
 class
   ( Abelian t,
-    Eq t,
-    PartialOrd t
```

Comment:
Reason for removing this? Do we no longer expect to be able to compare values?

Reply:
We compare values using pointwise. Exactly as it is described in the specification.
The operators (==), (<=), ... mean total order to programmers, so exporting the partial order equivalents is confusing.  Other than the use of (==), Which we define using pointwise. Which makes it clear that the default value of 0 plays an important role when comparing.   There is like only one use of comparison in the code base so complcate things with the partial order is just confusing. This is much simpler.